aws/request_pagination.go

```diff
@@ -42,24 +41,24 @@ type Pagination struct {
 	curPage interface{}
 }
 
-// HasNextPage will return true if Pagination is able to determine that the API
+// hasNextPage will return true if Pagination is able to determine that the API
 // operation has additional pages. False will be returned if there are no more
 // pages remaining.
 //
 // Will always return true if Next has not been called yet.
-func (p *Pagination) HasNextPage() bool {
+func (p *Pager) hasNextPage() bool {
```

Comment:
out of curiosity why not export this?

Reply:
With the new design, I couldn't think of a good reason to export it.